assets/scss/base/_table.scss

```diff
@@ -120,3 +120,7 @@ table {
     float: left;
   }
 }
+
+.table--font-small {
```

Comment:
is `small` the right name here considering the base for `table` is `core-16` this is normalizing the `table` contents to our base font size of `core-19` right?